### PR TITLE
Airlock construction

### PR DESCRIPTION
**Pull request by ShadowCommander**
## About the PR 

Adds a construction graph for the basic airlock.

I don't know what the normal timing is for these, so I just went with whatever.

**Changelog**

:cl:
- add: Added airlock construction and deconstruction

### CODE REVIEW

**Comment by ShadowCommander** [comment]
There is currently a physics collision bug where `Awake` gets set while `BodyType == BodyType.Static` when the player bumps the AirlockAssembly.

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -45,3 +45,6 @@
 
 - type: Tag
   id: Write
+
+- type: Tag
+  id: DoorElectronics
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:50]: Alphabetical plz

File: Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs [hunk 2]
```diff
@@ -638,6 +648,39 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
...
+        {
+            // Ensure that the construction component is aware of the board container.
+            if (Owner.TryGetComponent(out ConstructionComponent? construction))
+                construction.AddContainer("board");
+
+            // We don't do anything if this is null or empty.
+            if (string.IsNullOrEmpty(_boardPrototype))
+                return;
+
+            var container = ContainerHelpers.EnsureContainer<Container>(Owner, "board", out var existed);
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs:666]: There's a helper for this, Owner.EnsureContainer IIRC

File: Resources/Prototypes/Entities/Constructible/Doors/airlock_base.yml [hunk 3]
```diff
@@ -68,8 +69,57 @@
...
+  - type: Physics
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+          bounds: "-0.49,-0.49,0.49,0.49" # don't want this colliding with walls or they won't close
+      mask:
+      - MobImpassable
+      layer:
+      - MobImpassable
+      - VaultImpassable
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Constructible/Doors/airlock_base.yml:103]: Should have a mass ideally (defaults to 1)

File: Resources/Prototypes/Recipes/Construction/Graphs/airlock.yml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    - node: wired
+      entity: AirlockAssembly
+      edges:
+        - to: electronics
+          conditions:
+            - !type:EntityAnchored {}
+          steps:
+            - tag: DoorElectronics
+              store: board
+              name: "[color=cyan]door electronics[/color] circuit board"
```
Comment (metalgearsloth) [inline Resources/Prototypes/Recipes/Construction/Graphs/airlock.yml:53]: This shows up verbatim in-game.

**Review by metalgearsloth — COMMENTED** [review]
Just some things I noticed when trying in-game.

**Review by PaulRitter — DISMISSED** [review]

**Review by Zumorica — APPROVED** [review]
seems good

**Review by Zumorica — APPROVED** [review]

**Pull request merged**